src/boot/bootctl-uki.c

```diff
@@ -146,18 +147,36 @@ static int read_pe_section(FILE *uki, const struct PeSectionHeader *section,
 static void inspect_uki(FILE *uki, struct PeSectionHeader *sections, size_t scount) {
         _cleanup_free_ char *cmdline = NULL;
         _cleanup_free_ char *uname = NULL;
-        size_t idx;
+        _cleanup_free_ char *osrel = NULL;
+        size_t osrel_size, idx;
 
         if (find_pe_section(sections, scount, name_cmdline, sizeof(name_cmdline), &idx))
                 read_pe_section(uki, sections + idx, (void**)&cmdline, NULL);
 
         if (find_pe_section(sections, scount, name_uname, sizeof(name_uname), &idx))
                 read_pe_section(uki, sections + idx, (void**)&uname, NULL);
 
+        if (find_pe_section(sections, scount, name_osrel, sizeof(name_osrel), &idx))
+                read_pe_section(uki, sections + idx, (void**)&osrel, &osrel_size);
+
         if (cmdline)
                 printf("    Cmdline: %s\n", cmdline);
         if (uname)
                 printf("    Version: %s\n", uname);
+        if (osrel) {
+                _cleanup_fclose_ FILE *s = fmemopen(osrel, osrel_size, "r");
```

Comment:
if fmemopen fails (oom or such), parse_env_file will assert. Initialize to NULL and move the fmemopen down, and check for errors.